akka-docs-dev/rst/migration-guide-1.0-2.x.rst

```diff
@@ -48,7 +49,48 @@ Update procedure
 5. Repplace all uses of ``BidiFlow.apply()`` (Scala DSL) or ``BidiFlow.create()`` (Java DSL) when it converts two
```

Comment:
There's a

```
``Flow``s
```

in the live above, which AFAIR breaks rst rendering (must get a space after ` ane before `s` and it's OK)